ruma-events/src/key/verification/start.rs

```diff
@@ -29,34 +29,34 @@ pub enum StartEventContent {
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct MSasV1Content {
```

Comment:
Since this already has a constructor, can you make it `#[non_exhaustive]`, ideally as a `--fixup` commit? :)

Reply:
(see #14 for rationale)